src/utils/data.js

```diff
@@ -39,3 +39,15 @@ export async function getData() {
     }
   });
 }
+
+export async function getSearchData(str) {
+  try {
+    console.log("getting search data");
+    let res = await fetch(
+      `http://dev.virtualearth.net/REST/v1/Locations/${str}?key=Al2-1GXrd8GzwtSAID3J3LJn-flLLgNWzNtsT5nnSKW8dA2ClgaXXXMQR6WfE6wE`
```

Comment:
@jimboweb this is probably why the search feature isn't working in the Netlify preview. When sites are served using `https` calls to APIs that use `http` will be blocked.

I would also recommend stashing the API key in `src/utils/config` and importing it here. It's helpful to have all API & external service configuration in one place.